recipes/warcio/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "warcio" %}
+{% set version = "1.4.0" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "c0290ccd313e01058e9072e9928479bb45d9407bb3a76dce715e40470ba4f1c3" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  noarch: python
+  preserve_egg_dir: True
```

Comment:
Are we sure this is needed?

Reply:
It was generated by skeleton, so I've kept it. Not sure what the right test is for validating this?

Reply:
I *think* you only need this if you have non-`console_scripts` entry points, which warcio [doesn't](https://github.com/webrecorder/warcio/blob/594eb8c8b0fb35e03294635d23146160832b4371/setup.py#L41).